src/image-copy/azext_imagecopy/custom.py

```diff
@@ -16,7 +16,7 @@
 # pylint: disable=too-many-statements
 # pylint: disable=too-many-locals
 def imagecopy(source_resource_group_name, source_object_name, target_location,
-              target_resource_group_name, source_type='image', cleanup='false',
+              temporary_rg_name, target_resource_group_name, source_type='image', cleanup='false',
```

Comment:
1. I think it makes sense to have both target parameters together, so please move the new _temporary_rg_name_ parameter after _target_resource_group_name_
1. Shouldn't the new parameter have a default value set here too, like all others have?

Reply:
Sorry, not really sure what I did with 'approve these changes.' :) thought that was approving them for my PR. 

I believe it should have a default value as well as the other ones. Do I need to make these changes within my source and re-submit the PR or can it be changed within this view directly?

Reply:
You should make changes in your source code and push to you fork. The PR will be updated automatically without the need to re-submit it.

Reply:
BTW, looks like you also need to do a merge with the latest changes on the master branch